x-pack/plugins/reporting/server/export_types/csv_from_savedobject/types.d.ts

```diff
@@ -4,25 +4,26 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { JobParamPostPayload, ScheduledTaskParams, TimeRangeParams } from '../../types';
+import { TimeRangeParams } from '../../types';
 
 export interface FakeRequest {
-  headers: Record<string, unknown>;
-}
-
-export interface JobParamsPostPayloadPanelCsv extends JobParamPostPayload {
-  state?: any;
+  headers: Record<string, string>;
 }
 
 export interface JobParamsPanelCsv {
   savedObjectType: string;
   savedObjectId: string;
-  isImmediate: boolean;
-  panel?: SearchPanel;
```

Comment:
`panel` was optional because this interface was used for 2 different things. Now there is `jobParamsPanelCsv` with no panel field, and `JobPayloadPanelCsv` with all the original params, plus the panel field.